python/feathub/feature_views/feature_view.py

```diff
@@ -44,6 +46,12 @@ def __init__(
                          which refers to a feature in the source table.
         :param keep_source_fields: True iff all fields in the source table should be
                                    included in this table.
+        :param timestamp_field: Optional. Optional. If it is not None, it is the
```

Comment:
There are a couple of issues in this doc. Can you improve it?

Same for `timestamp_format`.